blackjack.el

```diff
@@ -1065,13 +1065,13 @@ Can be a single-character currency symbol such as \"$\", \"€\" or \"£\", or a
 (define-minor-mode blackjack-minor-mode
   "Blackjack minor mode.
 
-\\{blackjack-mode-map}"
+\\{blackjack-minor-mode-map}"
   :group 'blackjack
   :lighter " blackjack")
 
-(define-derived-mode blackjack-mode fundamental-mode "Blackjack"
+(define-derived-mode blackjack-mode special-mode "Blackjack"
```

Comment:
What does this do?

Reply:
Change parent mode to special-mode.

This seems like the right choice, referring to the Elisp manual, which says that
special-mode is the mode to inherit from if the buffer displays text generated
programmatically by Emacs and not representing a file.

Since special-mode automatically makes the buffer read-only, we no longer need to call `read-only-mode` explicitly.